src/main/webapp/dosave.jsp

```diff
@@ -461,4 +462,128 @@
 <%
         }
     } // ENDE if (speichern)
+
+if (form.equals("einzelbeleg")) {
+
+try{
+
+boolean hasZusatzNamenKommentar = EinzelbelegDB.hasZusatzNamenKommentar(String.valueOf(id));
+boolean hasLemma = EinzelbelegDB.hasLemmar(String.valueOf(id));
+
+Einzelbeleg lastEinzelbeleg = EinzelbelegDB.getById(id);
+belegform = lastEinzelbeleg.getBelegform(); // z.B. Sebastianus
+
+
+if (!hasZusatzNamenKommentar) { %>
+
+    <script>
+        $(function () {
+            var einzelbelegID = <%= id %>;
+
+            let ajaxUrl = '<%= Utils.getBaseUrl(request) %>/ajax';
```

Comment:
Just a thought... there are many of these cases where we build a similar URL... maybe we should add a function `Utils.getAjaxUrl(request)` that returns the full path to the ajax servlet

Reply:
done, oder findest du das besser: in der Util:

 public static String getMyUrl(HttpServletRequest request, String lastUrl) {
        return getBaseUrl(request) + "/" + lastUrl;
    }  
  und in der dosave.jsp:

let ajaxUrl = '<%= Utils.getMyUrl(request, "ajax") %>';


so kann man die funktion für verschiedene urls benutzen , im pull ist diese version nicht, da habe ich nur:  

 let ajaxUrl =  '<%= Utils.getAjaxUrl(request) %>';  mit der funktion: public static String getAjaxUrl(HttpServletRequest request) {
        return getBaseUrl(request) + "/ajax";
    }